Content.Server/GameObjects/Components/Arcade/BlockGameArcadeComponent.cs

```diff
@@ -538,36 +538,46 @@ private void InitializeNewBlock(BlockGamePiece piece)
 
             public void ProcessInput(BlockGamePlayerAction action)
             {
+                if (_running)
+                {
+                    switch (action)
+                    {
+                        case BlockGamePlayerAction.StartLeft:
+                            _leftPressed = true;
+                            break;
+                        case BlockGamePlayerAction.EndLeft:
+                            _leftPressed = false;
+                            break;
+                        case BlockGamePlayerAction.StartRight:
+                            _rightPressed = true;
+                            break;
+                        case BlockGamePlayerAction.EndRight:
+                            _rightPressed = false;
+                            break;
+                        case BlockGamePlayerAction.Rotate:
+                            TrySetRotation(Next(_currentRotation, false));
+                            break;
+                        case BlockGamePlayerAction.CounterRotate:
+                            TrySetRotation(Next(_currentRotation, true));
+                            break;
+                        case BlockGamePlayerAction.SoftdropStart:
+                            _softDropPressed = true;
+                            if (_accumulatedFieldFrameTime > Speed) _accumulatedFieldFrameTime = Speed; //to prevent jumps
+                            break;
+                        case BlockGamePlayerAction.SoftdropEnd:
+                            _softDropPressed = false;
+                            break;
```

Comment:
should also be executed when not running. else when you would pause pressing down, it would still softdrop on unpause, even if you stopped holding down